README.md

```diff
@@ -83,13 +83,7 @@ npm run lint:js -- --fix
 
 ### Check for broken links
 
-Build the site
```

Comment:
This step is not required anymore?

Reply:
No, it's managed by `"pretest:links": "npm run build"`